kqueen_ui/blueprints/ui/templates/ui/partial/selectform.html

```diff
@@ -16,7 +16,8 @@
         {% endif %}
       {% endif %}
     {% endfor %}
-    <input type="submit" value="Submit" class="btn btn-primary btn-sm">
+    <input type="button" name='cancel' value="Cancel" class="btn btn-default btn-sm"  onclick="window.location.href='{{ url_for('ui.index') }}';">
```

Comment:
Can you click around a bit and confirm that this template is only used for forms which are supposed to have cancel button leading to `ui.index`? It may be undesirable in some cases

Reply:
Currently it is used in cluster_create.html and provisioner_create.html, so it is fine